src/DependencyInjection/HostnetAssetExtension.php

```diff
@@ -48,6 +43,32 @@ public function load(array $configs, ContainerBuilder $container)
     {
         $config = $this->processConfiguration($this->getConfiguration($configs, $container), $configs);
 
+        // Create the node information
+        $node_executable = new Definition(Executable::class, [
+            $config['bin']['node'],
+            $config['bin']['node_modules'],
+        ]);
+        $node_executable->setPublic(false);
+
+        $container->setDefinition('hostnet_asset.node.executable', $node_executable);
+
+        $plugins  = [];
+        $built_in = [TsPlugin::class, LessPlugin::class, AngularPlugin::class];
+        foreach ($config['plugins'] as $name => $data) {
+            if (! $data) {
```

Comment:
How about `$settings` instead of `$data`, since we're talking about specific settings for the registered plugins.